track_odometry/src/track_odometry.cpp

```diff
@@ -79,10 +83,17 @@ geometry_msgs::Vector3 toVector3(const Eigen::Vector3f& a)
 class TrackOdometryNode
 {
 private:
+  using SyncPolicy =
+      message_filters::sync_policies::ApproximateTime<nav_msgs::Odometry, sensor_msgs::Imu>;
```

Comment:
Is the approximate synchronization OK for Odometry and IMU?
Interpolation would be the option as well.

Reply:
I agree that interpolation is better, but it requires additional implementation. (Is interpolation provided by message_filter?)
ApproximateTime is better than master branch, anyway.

Reply:
> Is interpolation provided by message_filter?

No, I don't think so. 

>  ApproximateTime is better than master branch, anyway.

Right. I'm OK to merge the PR if you want.

Reply:
@at-wat Can you open an Issue regarding to the interpolation?

Reply:
added to https://github.com/at-wat/neonavigation/issues/358